js/src/sdk/utils/errors/src/formatter.ts

```diff
@@ -44,22 +37,26 @@ export const getAPIErrorDetails = (
     case SDK_ERROR_CODES.BACKEND.SERVER_UNAVAILABLE:
     case SDK_ERROR_CODES.BACKEND.RATE_LIMIT:
       return {
-        message: `${predefinedError.message || axiosError.message} for ${axiosError.config.baseURL + axiosError.config.url}`,
-        description:
-          axiosError.response?.data?.message || predefinedError.description,
-        possibleFix:
-          predefinedError.possibleFix || defaultErrorDetails.possibleFix,
+        message: `${predefinedError.message || axiosError.message} for ${axiosError.config?.baseURL! + axiosError.config?.url!}`,
```

Comment:
Consider extracting the URL construction into a utility function to avoid repetition and potential errors. The pattern `${axiosError.config?.baseURL! + axiosError.config?.url!}` is used multiple times. Example:
```typescript
const getFullUrl = (config?: { baseURL?: string; url?: string }) => 
  config ? `${config.baseURL || ''}${config.url || ''}` : '';
```
This would make the code more maintainable and reduce the risk of null reference errors.